src/test/java/api/CirculationRulesAPITests.java

```diff
@@ -67,4 +79,16 @@ private String getRulesText() {
     return text;
   }
 
+  private void setRules(String rules) {
+    createPoliciesIfDoNotExist(rules);
+    circulationRulesFixture.updateCirculationRules(rules);
+  }
+
+  private void createPoliciesIfDoNotExist(String rules) {
+    loanPoliciesFixture.create(getPolicyFromRule(rules, "l"));
+    noticePoliciesFixture.create(getPolicyFromRule(rules, "n"));
+    requestPoliciesFixture.allowAllRequestPolicy(getPolicyFromRule(rules, "r"));
+    overdueFinePoliciesFixture.create(getPolicyFromRule(rules, "o"));
+    lostItemFeePoliciesFixture.create(getPolicyFromRule(rules, "i"));
+  }
```

Comment:
Personally, I would create the policies explicitly and then refer to them in the rules.

That might be a step too far from the approach taken in this work.

Reply:
Updated

Reply:
Thank you